Type Error when I try to set a breakpoint in a Drupal controller.

- [ERROR] {Sa 09 2018 18:11:31} An error occured: <class 'TypeError'>
Traceback (most recent call last):
  File "/home/k4v/.vim/bundle/vdebug/pythonx/vdebug/event.py", line 524, in dispatch_event
    Dispatcher.events[name](self.__session_handler).run(*args)
  File "/home/k4v/.vim/bundle/vdebug/pythonx/vdebug/event.py", line 262, in run
    if str(status) == "interactive":
TypeError: __str__ returned non-string (type NoneType)

--- a/python3/vdebug/event.py
+++ b/python3/vdebug/event.py
@@ -258,12 +258,18 @@ class WatchWindowContextChangeEvent(Event):
 class RefreshEvent(Event):
 
     def run(self, status):
-        if str(status) == "interactive":
+
+        status_str = str(status)
+
+        if not status_str:
+            return
+
+        if status_str == "interactive":
             self.ui.error("Debugger engine says it is in interactive mode,"
                           "which is not supported: closing connection")
             log.Log("closing connection because of interactive mode")
             self.session.close_connection()
-        elif str(status) in ("stopping", "stopped"):
+        elif status_str in ("stopping", "stopped"):
             self.ui.set_status("stopped")
             self.ui.say("Debugging session has ended")
             log.Log("closing connection because status is stopped")

